data/json/items/comestibles.json

```diff
@@ -12196,6 +12196,35 @@
     },
     {
         "type" : "COMESTIBLE",
+        "id" : "quesadilla_cheese",
+        "name" : "quesadilla",
+        "weight" : 116,
+        "color" : "brown",
+        "addiction_type" : "none",
+        "spoils_in" : 480,
+        "stim" : 0,
+        "container" : "wrapper",
+        "to_hit" : 0,
+        "comestible_type" : "FOOD",
+        "symbol" : "%",
+        "quench" : 0,
+        "healthy" : 1,
+        "addiction_potential" : 0,
+        "nutrition" : 33,
+        "description" : "A tortilla filled with cheese and lighty grilled.",
+        "price" : 500,
+        "material" : ["milk"],
+        "tool" : "null",
+        "volume" : 1,
+        "cutting" : 0,
```

Comment:
You may want to remove this line but it's not a blocker (related to your recent question in another PR).